sale_blanket_order/models/blanket_orders.py

```diff
@@ -89,8 +89,6 @@ def _compute_amount_all(self):
     payment_term_id = fields.Many2one(
         "account.payment.term",
         string="Payment Terms",
-        readonly=True,
-        states={"draft": [("readonly", False)]},
```

Comment:
Should payment terms change after confirmation ? Strange

Reply:
@rousseldenis  Edit Payment Terms while a Blanket Order is open should not be a problem because the sale orders you create will have this Payment Term. 

You can agree with your client to change the Payment Terms and once you change it in the Blanket Order will affect the next Sale Orders.